bin/update_cucumber

```diff
@@ -20,8 +20,8 @@ if (ini_get('zend.assertions') !== '1') {
 
 $updater = new
 /**
- * @phpstan-type CurrentVersionArray array{tag_name: string, hash: string, composer_version: string}
- * @phpstan-type GitHubReleaseArray array{tag_name: string, hash: string, html_url: string, changelog: string}
+ * @phpstan-type TCurrentVersionArray array{tag_name: string, hash: string, composer_version: string}
+ * @phpstan-type TGitHubReleaseArray array{tag_name: string, hash: string, html_url: string, changelog: string}
```

Comment:
@acoulton I decided to keep the same convention of prefixing the alias types with a 'T' in some languages this convention is used to denote 'types' (as opposed to classes). I've been doing that to avoid conflicts with real PHP classes (although the 'Array' suffix would have adequately solved that in this case - with added readability).